<launch>

    <include file="$(find fugu_configurations)/launch/camera_flex_narrow_image_proc.launch" />

    <node pkg="nodelet" name="stereo_feature_extractor" output="screen"
          ns="/forward_stereo_camera" type="nodelet" 
          args="standalone stereo_feature_extraction/stereo_feature_extractor" />

    <node pkg="object_detection" name="motion_estimator" 
          type="movement_calculator_node" output="screen">
        <remap from="stereo_features" to="/forward_stereo_camera/stereo_features"/>
        <param name="ransac_threshold" value="0.2" />
    </node>

    <node pkg="tf" name="world_camera_tf_publisher" 
          type="static_transform_publisher"
          args="1 0 0 0 1.507 0 world stereo_camera 200"/>

    <node pkg="rviz" name="rviz" type="rviz" />
</launch>
